ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java

```diff
@@ -1403,4 +1400,15 @@ public static Table tableIfExists(ImportTableDesc tblDesc, Hive db) throws HiveE
     }
   }
 
+  private static Path getTableDataLocation(Warehouse wh, ImportTableDesc tblDesc,
+      EximUtil.SemanticAnalyzerWrapperContext x) throws HiveException, MetaException {
+    Table translatedTable =
+        x.getHive().getTranslateTableDryrun(tblDesc.toTable(x.getConf()).getTTable());
```

Comment:
Any chance that SemanticAnalyzerWrapperContext can be null?

Reply:
no, we always create it at ImportSemanticAnalyzer#L170